README.md

```diff
@@ -2,21 +2,20 @@
 
 # OPTIMADE Providers Index Meta-Database
 
-The list of OPTIMADE providers keeps track of all reserved database-specific prefixes and the URLs to the index databases of all OPTIMADE database providers that participate in the [OPTIMADE network](https://www.optimade.org/).
+The list of providers is published as an [OPTIMADE](https://www.optimade.org/) Index Meta-Database here: [https://providers.optimade.org/](https://providers.optimade.org/)
 
-The list of providers is published in the form of a statically hosted OPTIMADE Index Meta-Database here:
+You can obtain the list of providers in a machine-readable format following the [OPTIMADE specification for Index Meta-Databases](https://github.com/Materials-Consortia/OPTIMADE/blob/develop/optimade.rst#32index-meta-database) by using the following URL:
 
-- [https://providers.optimade.org/](https://providers.optimade.org/)
+- [https://providers.optimade.org/v1/links](https://providers.optimade.org/v1/links)
 
-If you specifically seek the current list of providers for the latest version of the OPTIMADE specification, you can access it at this URL:
+where `v1` is currently the latest version and can be replaced with any major version name of the OPTIMADE specification.
 
-- [https://providers.optimade.org/providers.json](https://providers.optimade.org/providers.json)
+For convenience, you can also access the most recent list of providers for the latest version of the OPTIMADE specification directly at this URL:
```

Comment:
I have noted a possible misunderstanding here, concerning what "latest version" refers to. I could interpret this in two ways:
1. "Latest version" refers to the file version, which acts as a sort of "backend"/db for the "view" on it through the `/links` endpoint, which has a different version, namely, the version of the OPTIMADE API.
1. "Latest version` refers to the OPTIMADE API version.

I think, I would just add a note here stating that this file's "response format" complies to the latest major version of the OPTIMADE API, and that's clear enough, right?

Reply:
"... in the format mandated by the latest version of the OPTIMADE specification directly at this URL"